zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/utils/AWSSearchDBExtension.java

```diff
@@ -50,26 +66,41 @@ public void beforeEach(final ExtensionContext context) throws Exception {
         .createIndex(PROCESS_INDEX, new IndexSettings());
   }
 
+  /**
+   * @see SearchDBExtension#objectMapper()
```

Comment:
Here and other methods the same, we should use inheritDoc instead

Reply:
Done